### DISCUSSION TITLE
Only notifications when updates have taken place

### DISCUSSION THREAD

**Discussion by Jonathan137** [containrrr/watchtower#2024] [General] [answered]
I'm using Watchtower combined with Gotify
I use these flags in my compose file:
      - WATCHTOWER_NO_STARTUP_MESSAGE=false
      - WATCHTOWER_NOTIFICATIONS=gotify
      - WATCHTOWER_NOTIFICATION_GOTIFY_URL=http.....
      - WATCHTOWER_NOTIFICATION_GOTIFY_TOKEN=.......
      - WATCHTOWER_NOTIFICATIONS_HOSTNAME=docker
     
Since I have 8 docker servers running, each 24 hours I get a lot of Watchtower notifications.
I'm only interested to see if images are updated.

Is it possible to prevent sending notifications when none of the images on the docker server are updated?

**Comment by chiragkrishna** [comment, marked as answer]
Yes use [Report templates](https://containrrr.dev/watchtower/notifications/#report_templates), you can check this out https://github.com/containrrr/watchtower/discussions/2022

**Reply by Jonathan137** [reply]
Thanks, will try
